SIPS/sip-29.md

```diff
@@ -35,85 +35,140 @@ in uppercase in this document are to be interpreted as described in [RFC
 2. Any time an asset needs to be identified, it MUST use the [CAIP-19][caip-19]
 representation.
 
-### Snap Assets API
+### Snap Manifest
+
+This SIP introduces a new permission named `endowment:assets`.
+This permission grants a Snap the ability to provide asset information to the client.
+
+This permission is specified as follows in `snap.manifest.json` files:
+
+```json
+{
+  "initialPermissions": {
+    "endowment:assets": {}
```

Comment:
I'm wondering if a caveat for the supported chains here would be useful.

Reply:
```json
{
  "initialPermissions": {
    "endowment:assets": {
     "chains": [
         "foo:bar"
       ]
    }
}
```

Reply:
Probably yeah - could work as a way to reduce calls to Snaps that know about assets on unwanted chains